kolibri/core/device/migrations/0005_auto_20191203_0951.py

```diff
@@ -0,0 +1,62 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.23 on 2019-12-03 17:51
+from __future__ import unicode_literals
+
+from django.db import migrations
+from django.db import models
+
+
+def migrate_allow_guest_access(apps, schema_editor):
+    FacilityDataset = apps.get_model("kolibriauth", "FacilityDataset")
+    DeviceSettings = apps.get_model("device", "DeviceSettings")
+
+    allow_guest_access = (
+        FacilityDataset.objects.filter(allow_guest_access=False).count() <= 0
```

Comment:
Could we use the default facility here, by default, then just aggregate if it doesn't exist?